facade/src/main/scala/reactST/reactTable/Hooks.scala

```diff
@@ -0,0 +1,36 @@
+package reactST.reactTable
+
+import reactST.reactTable.mod._
+import scala.scalajs.js
+import scala.scalajs.js.`|`
+import scala.scalajs.js.annotation.{ JSBracketAccess, JSGlobal, JSGlobalScope, JSImport, JSName }
+
+object Hooks {
```

Comment:
The facades that the newer scalablytyped generates no longer work as hooks, so I had to create this. This does not affect users of the library.